### ISSUE TITLE
Importing single recipe from allrecipes.com does not import ingredients locally but works in demo

### ISSUE THREAD

**Issue by lateplate** [TandoorRecipes/recipes#3228]
### Tandoor Version

1.5.18

### Setup

Docker / Docker-Compose

### Reverse Proxy

No reverse proxy

### Bug description

Hi, this one is a bit weird, but when I try to import any recipe into my self hosted version from allrecipes.com, the ingredients are not imported (the instructions are fine). But when I use the exact same url on the demo version of Tandoor, the ingredients are imported just fine. Example recipe: https://www.allrecipes.com/recipe/256825/warm-berry-compote/

Not a particularly complex setup – just using on local network via direct IP access. No reverse proxy or server in front of the webapp. Pretty straight Docker Compose setup, with maybe the only minor diff is that I named the containers.

It's probably some silly misconfiguration on my part, but I read through all the env variables and settings and don't see anything that would impact that. The logs didn't show any outright errors.

Happy to provide any debug info or logs if there are specific ones that might help. Thanks!

### Relevant logs

```shell
DEBUG=0

# random secret key, use for example `base64 /dev/urandom | head -c50` to generate one
SECRET_KEY=<redacted>

# allowed hosts (see documentation), should be set to your hostname(s) but might be * (default) for some proxies/providers
ALLOWED_HOSTS=*

# add only a database password if you want to run with the default postgres, otherwise change settings accordingly
DB_ENGINE=django.db.backends.postgresql
POSTGRES_HOST=db_recipes
POSTGRES_DB=djangodb
POSTGRES_PORT=5432
POSTGRES_USER=djangouser
POSTGRES_PASSWORD=<redacted>

---

services:
  db_recipes:
    container_name: tandoor-postgres
    restart: always
    image: postgres:16-alpine
    volumes:
      - ./postgresql:/var/lib/postgresql/data
    env_file:
      - ./.env

  web_recipes:
    container_name: tandoor
    restart: always
    image: vabene1111/recipes:latest
    # image: vabene1111/recipes:develop
    env_file:
      - ./.env
    ports:
      - 8383:8080
    volumes:
      - ./static:/opt/recipes/staticfiles
      - ./nginx:/opt/recipes/nginx/conf.d
      - ./media:/opt/recipes/mediafiles
    depends_on:
      - db_recipes

---

Gunicorn Media: False
Sqlite:  False 
        PostgreSQL: 16 
        Debug: False

SERVER_PROTOCOL:HTTP/1.1
REMOTE_ADDR:::ffff:192.168.1.111
SERVER_PORT:8080

HTTP_HOST:192.168.1.4:8383
HTTP_USER_AGENT:Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:128.0) Gecko/20100101 Firefox/128.0
HTTP_ACCEPT:text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/png,image/svg+xml,*/*;q=0.8
HTTP_ACCEPT_LANGUAGE:en-US,en;q=0.5
HTTP_ACCEPT_ENCODING:gzip, deflate
HTTP_REFERER:http://192.168.1.4:8383/system/
HTTP_CONNECTION:keep-alive
HTTP_COOKIE:nd-player-64616e6b696d=3628f986-6c94-4bb0-8117-2aa9d630fd2e; csrftoken=UifPEnmzNfje9dSvCUVlN5wvtYd1WnLI; sessionid=uwbkzjz398nqr4mtunrsnlr1kp7yuyqi
HTTP_UPGRADE_INSECURE_REQUESTS:1
HTTP_PRIORITY:u=0, i

wsgi.errors:<gunicorn.http.wsgi.WSGIErrorsWrapper object at 0x7ac7d2179b70>
wsgi.version:(1, 0)
wsgi.multithread:True
wsgi.multiprocess:True
wsgi.run_once:False
wsgi.file_wrapper:
wsgi.input_terminated:True
wsgi.input:<gunicorn.http.body.Body object at 0x7ac7d2179c40>
wsgi.url_scheme:http
```

**Comment by smilerz** [comment]
fixed in #3227

**Comment by lateplate** [comment]
Thank you! 🥳
